### ISSUE TITLE
can not detected the comment end of the macro

### ISSUE THREAD

**Issue by heartacker** [microsoft/vscode-cpptools#8429]
Bug type: Language Service

**Describe the bug**
版本: 1.63.0-insider (system setup)
提交: bedf867b5b02c1c800fbaf4d6ce09cefbafa1592
日期: 2021-11-18T05:14:59.750Z
Electron: 13.5.2
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Windows_NT x64 10.0.14393
cpptools: 1.7.1

**Steps to reproduce**
1. creat a define
```c
#define _ANA_LPOSC_CFG_RG_MCU_LPOSC_CALI_Msk    0xFC000000 /*!< [31:26],charging cap calibration */
```
2. hover the define
3. no comment description

![image](https://user-images.githubusercontent.com/1876302/142756540-9bab7f7c-7300-4600-beb0-7eecc03c0c71.png)

**Expected behavior**

show me the comment

**Code sample and logs**

- Code sample

- Configurations in `c_cpp_properties.json`

- Logs from running `C/C++: Log Diagnostics` from the VS Code command palette

- Logs from [the language server logging](https://code.visualstudio.com/docs/cpp/enable-logging-cpp#_enable-logging-for-the-language-server)

**Screenshots**

**Comment by sean-mcmanus** [comment]
This seems like a duplicate of https://github.com/microsoft/vscode-cpptools/issues/8320 .

**Comment by sean-mcmanus** [comment]
Fixed with 1.11.3 (pre-release): https://github.com/microsoft/vscode-cpptools/releases/tag/v1.11.3
